python/pylibcugraph/pylibcugraph/graphs.pyx

```diff
@@ -286,93 +322,141 @@ cdef class MGGraph(_GPUGraph):
         Device array containing the edge types of each directed edge.  Must
         match the ordering of the src/dst/edge_id arrays.  Optional (may be
         null).  If provided, edge_id_array must be provided.
+    
+    drop_self_loops : bool, optional (default='False')
+        If true, drop any self loops that exist in the provided edge list.
+    
+    drop_multi_edges: bool, optional (default='False')
+        If true, drop any multi edges that exist in the provided edge list
     """
     def __cinit__(self,
                   ResourceHandle resource_handle,
                   GraphProperties graph_properties,
                   src_array,
                   dst_array,
+                  vertices_array=None,
                   weight_array=None,
                   store_transposed=False,
-                  num_edges=-1,
-                  do_expensive_check=False,
+                  size_t num_arrays=1, # default value to not break users
+                  do_expensive_check=True,
                   edge_id_array=None,
-                  edge_type_array=None):
+                  edge_type_array=None,
+                  drop_self_loops=False,
+                  drop_multi_edges=False):
```

Comment:
same with these: can these be arranged so the new params are last so users with position-only args aren't broken?